sdk/src/main/java/com/ifountain/opsgenie/client/model/BaseUserRequest.java

```diff
@@ -1,7 +1,11 @@
 package com.ifountain.opsgenie.client.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
 public abstract class BaseUserRequest<T extends BaseResponse, K extends BaseUserRequest> extends BaseRequest<T, K> {
+    @JsonIgnore
```

Comment:
What does the `JsonIgnore` here signify? Is `username`, `userId` not mandatory in requests?